_examples/spreadsheet/flatten/main.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 FoxyUtils ehf. All rights reserved.
+package main
```

Comment:
Add a description, e.g.
// This example demonstrates flattening all formulas from an input Excel file and outputs the flattened values to a new xlsx.